spec/presenters/summary/sections/other_outgoings_details_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+describe Summary::Sections::OtherOutgoingsDetails do
+  subject { described_class.new(crime_application) }
+
+  let(:crime_application) do
+    instance_double(
+      CrimeApplication,
+      to_param: '12345',
+      outgoings: outgoings,
+    )
+  end
+
+  let(:outgoings) do
+    instance_double(
+      Outgoings,
+      income_tax_rate_above_threshold: 'no',
+      outgoings_more_than_income: 'yes',
+      how_manage: 'An example of how they manage'
+    )
+  end
+
+  describe '#name' do
+    it { expect(subject.name).to eq(:other_outgoings_details) }
+  end
+
+  describe '#show?' do
+    context 'when there is an outgoings_details' do
+      it 'shows this section' do
+        expect(subject.show?).to be(true)
+      end
+    end
+
+    context 'when there is no outgoings_details' do
+      let(:outgoings) { nil }
+
+      it 'does not show this section' do
+        expect(subject.show?).to be(false)
+      end
+    end
+  end
+
+  describe '#answers' do
+    let(:answers) { subject.answers }
+
+    context 'when there are outgoings details' do
+      it 'has the correct rows' do
```

Comment:
Only thing I'm thinking is if we could write amethod for these so we aren't just calling the same expects with only slightly varing values

Think I did something like this for the IncomeDetails screens.

Reply:
Aha, yep!
Will do 👍